editor/src/components/common/shared-strategies/convert-to-flex-strategy.ts

```diff
@@ -338,3 +357,151 @@ function guessAlignItems(
   // fallback: null, which implicitly means a default of flex-start
   return null
 }
+
+function isElementNonDOMElement(
+  metadata: ElementInstanceMetadataMap,
+  allElementProps: AllElementProps,
+  elementPath: ElementPath,
+): boolean {
+  const contentAffectingType = getElementContentAffectingType(
+    metadata,
+    allElementProps,
+    elementPath,
+  )
+  return AllContentAffectingNonDomElementTypes.some((type) => contentAffectingType === type)
+}
+
+interface NonDOMElementWithLeaves {
+  group: ElementPath // path to a non-DOM element
```

Comment:
This should probably be `element` (or `nonDOMElement`) rather than `group`

Reply:
renamed in https://github.com/concrete-utopia/utopia/pull/3557/commits/cfa51fc45ccc17f03e5429a1f05327f498803b9e